torchcomplex/nn/functional.py

```diff
@@ -63,20 +63,37 @@ def _fcaller(funtinal_handle, *args):
             b_r = None
             b_i = None
     
+    bias_added = True
+
     # Perform complex valued convolution
     if type(args[0]) is tuple: #only incase of bilinear
         MrKr = funtinal_handle(inp1_r, inp2_r, w_r, b_r, *args[3:]) #Real Feature Maps *(conv) Real Kernels
         MiKi = funtinal_handle(inp1_i, inp2_i, w_i, b_i, *args[3:]) #Imaginary Feature Maps * Imaginary Kernels
         MrKi = funtinal_handle(inp1_r, inp2_r, w_i, b_i, *args[3:]) #Real Feature Maps * Imaginary Kernels
         MiKr = funtinal_handle(inp1_i, inp2_i, w_r, b_r, *args[3:]) #Imaginary Feature Maps * Real Kernels
     else:
-        MrKr = funtinal_handle(inp_r, w_r, b_r, *args[3:]) #Real Feature Maps *(conv) Real Kernels
-        MiKi = funtinal_handle(inp_i, w_i, b_i, *args[3:]) #Imaginary Feature Maps * Imaginary Kernels
-        MrKi = funtinal_handle(inp_r, w_i, b_i, *args[3:]) #Real Feature Maps * Imaginary Kernels
-        MiKr = funtinal_handle(inp_i, w_r, b_r, *args[3:]) #Imaginary Feature Maps * Real Kernels
+        if args[6] == 1: #groups
+            bias_added = False
+            n_out = int(w_r.shape[0])
+            w     = torch.cat([w_r, w_i], dim=0)
+            MrKri = funtinal_handle(inp_r, w, None, *args[3:])
+            MiKri = funtinal_handle(inp_i, w, None, *args[3:])
+            MrKr,MrKi = MrKri[:, :n_out], MrKri[:, n_out:]
+            MiKr,MiKi = MiKri[:, :n_out], MiKri[:, n_out:]
+        else:
+            MrKr = funtinal_handle(inp_r, w_r, b_r, *args[3:]) #Real Feature Maps *(conv) Real Kernels
+            MiKi = funtinal_handle(inp_i, w_i, b_i, *args[3:]) #Imaginary Feature Maps * Imaginary Kernels
+            MrKi = funtinal_handle(inp_r, w_i, b_i, *args[3:]) #Real Feature Maps * Imaginary Kernels
+            MiKr = funtinal_handle(inp_i, w_r, b_r, *args[3:]) #Imaginary Feature Maps * Real Kernels
+
     real = MrKr - MiKi
     imag = MrKi + MiKr
     out = torch.view_as_complex(torch.stack((real,imag),dim=-1))
+
+    if not bias_added and b_r is not None and b_ri is not None:
```

Comment:
I've just noticed there is a mistake, it should be:
```b_i``` not ```b_ri```

Reply:
@soumickmj do you mind just committing to master rather than setting up another PR?